src/test/emulators/auth/oob.spec.ts

```diff
@@ -357,4 +358,41 @@ describeAuthEmulator("accounts:sendOobCode", ({ authApi, getClock }) => {
         expect(res.body.error).to.have.property("message").equals("PASSWORD_LOGIN_DISABLED");
       });
   });
+
+  it("should error when sending a password reset to non-existent user with improved email privacy disabled", async () => {
+    const user = { email: "alice@example.com", password: "notasecret" };
+    await authApi()
+      .post("/identitytoolkit.googleapis.com/v1/accounts:sendOobCode")
+      .set("Authorization", "Bearer owner")
+      .send({ email: user.email, requestType: "PASSWORD_RESET", returnOobLink: true })
+      .then((res) => {
+        expectStatusCode(400, res);
+        expect(res.body.error).to.have.property("message").equals("EMAIL_NOT_FOUND");
+      });
+  });
+
+  it("should return email when sending a password reset to non-existent user with improved email privacy enabled", async () => {
```

Comment:
This email should be sent only when the user exists. See https://cloud.google.com/identity-platform/docs/admin/email-enumeration-protection#overview

Reply:
Just to verify. Are you referring to this line “When you make a password reset request, a verification email is sent only if the email address exists”. If so, I think the name of the test case I wrote may be a little bit misleading.

In this case I’m referring to the email address to be returned in the api response. When an email does not exists, a password reset email would not be sent, just that the api response would look like
```json
{
    "kind": "identitytoolkit#GetOobConfirmationCodeResponse",
    "email": "non_existent_email@fake.fake"
}
```

Should I leave a comment in the test case describing this? Or just change the test name to “should return email address...”

LMK in case I misunderstood anything.

Reply:
Thanks for the clarification. I think "should return email address.." sounds more accurate.

Reply:
Thanks, updated the test title.